strawberry/types/info.py

```diff
@@ -113,6 +113,11 @@ def context(self) -> ContextType:
         """The context passed to the query execution."""
         return self._raw_info.context
 
+    @property
+    def input_extensions(self) -> dict[str, Any]:
```

Comment:
@patrick91 First of all, thanks a lot for taking the time to do it! I tested it in one of our APIs, the data does go through all the way! However it is returned as a string and not as a dict, so we need to `json.loads` the data before we return it from this function. The [spec](https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md#request-parameters) states that the parameter is expected to be a map so parsing it should be safe to do.

Reply:
@patrick91 actually nvm I was wrong, I was over-escaping the input, totally my bad. The implementation is actually correct!

Reply:
@omarzouk awesome! I'll release this after we fix the unrelated issue with GlobalID :D